.idea/libraries/Maven__de_flapdoodle_embed_de_flapdoodle_embed_mongo_2_2_0.xml

```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```

Comment:
I would include .idea folder in .gitignore since those files are not supposed to land on GitHub
maybe check if .gitignore is on correct level

Reply:
Thanks! .gitignore is currently missing! Will update in no time :)